pages/common/nvm.fish.md

```diff
@@ -0,0 +1,25 @@
+# nvm
+
+> Install, uninstall, or switch between Node.js versions under the `fish` shell.
+> Supports version numbers like "12.8" or "v16.13.1", and labels like "stable", "system", etc.
+> More information: <https://github.com/jorgebucaran/nvm.fish>.
+
+- Install a specific version of Node.js:
+
+`nvm install {{node_version}}`
+
+- Use a specific version of Node.js in the current shell:
+
+`nvm use {{node_version}}`
+
+- Set the default Node.js version:
+
+`set nvm_default_version {{node_version}}`
```

Comment:
Is this a typo? Should this be prefixed by `nvm` at all, or is this essentially updating an environment variable that `nvm` (fish) reads?

Reply:
Yes it uses the Fish' builtin `set` command to set the default version.